resources/META-INF/plugin.xml

```diff
@@ -172,8 +174,14 @@
     <!-- configurationType implementation="io.flutter.run.bazel.FlutterBazelRunConfigurationType"/>
     <programRunner implementation="io.flutter.run.bazel.FlutterBazelRunner"/ -->
 
+    <!-- IDEA only -->
     <moduleType id="FLUTTER_MODULE_TYPE" implementationClass="io.flutter.module.FlutterModuleType"/>
```

Comment:
sgtm, will move the `FlutterModuleConfigurationEditorProvider`. Off-hand, do you know why the four `editorNotificationProvider` are in the IDEA-only file?